client.go

```diff
@@ -67,14 +65,20 @@ func (c *Client) sendToStatsd(in chan *logMetrics) {
 	}
 }
 
-func (c *Client) sendRouterMsg(data *logMetrics) {
-
-	tags := *data.tags
-	for _, mk := range routerMetricsKeys {
-		if v, ok := data.metrics[mk]; ok {
+func (c *Client) extractTags(tags []string, permittedTags []string, metrics map[string]logValue) []string {
+	for _, mk := range permittedTags {
```

Comment:
Do I miss something here? We get `tags` and `permittedTags` as param, if the tag is not in the map of excluded tags, we append it if there is any metric and return the `tags` slice. So, if I send, `[]string { "a", "b", "c" }` as `tags` and `[]string{"b", "c"}` as permitted tags, it will append b and c with metric value to the end of `tags` slice and at the end of loop `tags` slice will have value like `{"a", "b", "c", "b:1",  "c:2" }`. I guess we want to just have `{"b:1",  "c:2" }`.